src/subplots.ts

```diff
@@ -2045,3 +2075,78 @@ export class Histogram extends Frame {
     return [mouse3X, mouse3Y];
   }
 }
+
+
+export class newScatter extends Frame {
+  public points: newPoint2D[] = [];
+  public tooltipAttr: string[];
+  readonly pointsColorFill: string = 'hsl(203, 90%, 85%)';
+  readonly pointsColorStroke: string = 'hsl(0, 0%, 0%)';
+  constructor(
+    public data: any,
+    public width: number,
+    public height: number,
+    public buttons_ON: boolean,
+    public X: number,
+    public Y: number,
+    public canvas_id: string,
+    public is_in_multiplot: boolean = false
+    ) {
+      super(data, width, height, buttons_ON, X, Y, canvas_id, is_in_multiplot);
+      if (!data.tooltip) {this.tooltipAttr = Array.from(this.features.keys()) }
+      else this.tooltipAttr = data.tooltip.attribute;
+    }
+
+  private computePoints(): newPoint2D[] {
+    const numericVectorX = this.axes[0].stringsToValues(this.features.get(this.xFeature));
+    const numericVectorY = this.axes[1].stringsToValues(this.features.get(this.yFeature));
+    const points: newPoint2D[] = [];
+    const pointsHTMatrix = this.getPointsHTMatrix(numericVectorX, numericVectorY);
+    const coordsMatrix = this.projectPoints(pointsHTMatrix);
+    for (let index = 0 ; index < numericVectorX.length ; index++) {
+      let newPoint = new newPoint2D(coordsMatrix.get(0, index),  coordsMatrix.get(1, index), 5, "circle", undefined, "hsl(203, 90%, 85%)");
+      if (this.hoveredIndices[index]) {newPoint.isHovered = true};
```

Comment:
Why not `newPoint.isSomethinged = this.somethingedIndices[index]` ?

Reply:
Done